examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,205 @@
     "tags": []
```

Comment:
<p>Let's use 100 shots for now. This will be enough to demonstrate, and not cost too much.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>